### ISSUE TITLE
TWE: Project Board Migration

### ISSUE THREAD

**Issue by KC-skc** [hackforla/internship#597]
### Overview
We need to prepare the Project Boards for the forced migration that GitHub is doing in August, so that it is already working well for the team by then.

### Action Items
- [x]  copy this template to a comment below
   ````
   ### Text from project board card
   ```
   [INSERT MARKDOWN HERE]
   ```
   - [ ] Determine where this item goes on the wiki (page and section)
   - [ ] copy item to wiki
   - [ ] hide this comment when completed.
   ````
- [x] copy markdown from a card in the "Start Here" column of the Project Board into your new comment where it says `[INSERT MARKDOWN HERE]
- [x] delete card from project board. (KC Archived card rather than delete.)

### Resources/Instructions
Classic Project board URL: https://github.com/hackforla/internship/projects/1
New Project board URL: https://github.com/orgs/hackforla/projects/64/

**Comment by KC-skc** [comment]
### Text from project board card
```
[INSERT MARKDOWN HERE]
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
All Team Meeting Times | [Agendas](https://github.com/hackforla/internship/projects/1?card_filter_query=agenda)


Thursday: 10 am - 11 am PT

PM meetings

Monday: 10 am - 11 am PT

Wednesday: 10 am - 11 am PT

Thursday: 9:30 am - 10 am PT

https://github.com/hackforla/internship/projects/1#card-65476150
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
#### Understand the Project
- [Internship Wiki](https://github.com/hackforla/internship/wiki)
- [One sheet - WIP](https://docs.google.com/document/d/1vovX2o0qiw4z9BUrH1Wb5RL9AzaveguQ1KMZ27nj11w/edit)

#### Team Links
- [Google Shared Drive](https://drive.google.com/drive/folders/1z3I2SLQzYWBHIypNZO-sTdSO0A8c5pvt?usp=sharing)
- [Internship AMA FAQ](https://docs.google.com/document/d/138TuoRoHN9S3IhbklFjge9QO4hBV2-eKGR0yR7CttzY/edit)
- [Intern Onboarding Deck](https://docs.google.com/presentation/d/1w9086NcRYTGQeTjbp6F0F2YDVSSVDkCiPDkCYxuOloE/edit#slide=id.gd0de62c6b7_1_0)
- [Linkedin Workshop Deck - WIP](https://docs.google.com/presentation/d/1qEnWq4rPxZm0ZtEsa1T0HoNpjuLxvJU2SlweiLYugwo/edit#slide=id.ge1b9cfd85e_1_138)
- [Roll Call/Hours/Onboarding status](https://docs.google.com/spreadsheets/d/11KC6P07dr_DEIq5lGZ_vqyakq-rfOEBSTqqduFLiVbs/edit)

#### Useful Hack for LA docs
- [Kanban Guide](https://docs.google.com/document/d/1ghc75Gio-Hfs_rpdSozOMmmdpdzFbze5ElLTvbrIS1U/edit#heading=h.nl3p4nf4eqb4)
https://github.com/hackforla/internship/projects/1#card-63275987
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
#### Understand the Project
- [Internship Wiki](https://github.com/hackforla/internship/wiki)
- [One sheet - WIP](https://docs.google.com/document/d/1vovX2o0qiw4z9BUrH1Wb5RL9AzaveguQ1KMZ27nj11w/edit)

#### Team Links
- [Google Shared Drive](https://drive.google.com/drive/folders/1z3I2SLQzYWBHIypNZO-sTdSO0A8c5pvt?usp=sharing)
- [Internship AMA FAQ](https://docs.google.com/document/d/138TuoRoHN9S3IhbklFjge9QO4hBV2-eKGR0yR7CttzY/edit)
- [Intern Onboarding Deck](https://docs.google.com/presentation/d/1w9086NcRYTGQeTjbp6F0F2YDVSSVDkCiPDkCYxuOloE/edit#slide=id.gd0de62c6b7_1_0)
- [Linkedin Workshop Deck - WIP](https://docs.google.com/presentation/d/1qEnWq4rPxZm0ZtEsa1T0HoNpjuLxvJU2SlweiLYugwo/edit#slide=id.ge1b9cfd85e_1_138)
- [Roll Call/Hours/Onboarding status](https://docs.google.com/spreadsheets/d/11KC6P07dr_DEIq5lGZ_vqyakq-rfOEBSTqqduFLiVbs/edit)

#### Useful Hack for LA docs
- [Kanban Guide](https://docs.google.com/document/d/1ghc75Gio-Hfs_rpdSozOMmmdpdzFbze5ElLTvbrIS1U/edit#heading=h.nl3p4nf4eqb4)

https://github.com/hackforla/internship/projects/1#card-63275987
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
Icebox = issues identified that are going to get addressed, but are not being started yet because of other more pressing issues.  Or issues that are not fully developed.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
New Issue Approval = When you create a new issue, please add it here so that it can be reviewed for completeness before its moved to the icebox or prioritized backlog. 🙏
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
Prioritized Backlog = features that are ready to work and a priority.  This column should be prioritized by most important first.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
In Progress = tasks that are actively being worked on by an individual team member and have been assigned to them in the issue.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by KC-skc** [comment]
### Text from project board card
```
Questions/Review = Issues that you are blocked on because you need an answer from another team member or other type of question/review for our team meeting.
```
- [ ] Determine where this item goes on the wiki (page and section)
- [ ] copy item to wiki
- [ ] hide this comment when completed.

**Comment by ExperimentsInHonesty** [comment]
We need to update any pages that have links to the old board on them.  Start here
https://github.com/hackforla/internship/wiki/Joining-the-Team

**Comment by KC-skc** [comment]
@ExperimentsInHonesty I've reviewed the wiki manually and didn't see too many references other than the page you linked above. Is there any other way to make sure? 

Also, in doing a review, I found a few pages in the wiki that requested permissions (which I should have access already but still asking for permission?) and some pages that were broken/deleted. What's the best way to address these?

Example 1: 
https://docs.google.com/document/d/1tYJVRJHQ9juQC3bNMtwqt6Z8y1BYLfrtp6ejZI9TCF4/edit
![Image](https://github.com/hackforla/internship/assets/148172362/daf1f44f-9a19-469b-8021-e0e9933f6870)

Example 2: 
https://docs.google.com/document/d/11Fe7mNdmPBP5bD_yLJ1C0_I1TmoK47AuHHrdhdDyWCs/edit#heading=h.nl3p4nf4eqb4
![Image](https://github.com/hackforla/internship/assets/148172362/f823ba67-aaf6-4b5c-96e7-a5a1b4601408)

**Comment by KC-skc** [comment]
Review of Wiki complete. Closing issue.
